### PR TITLE
Readiness probe for sentinel and exporter

### PR DESCRIPTION
**Pull request by michael-michalski**
Signed-off-by: Michael Michalski <michaelmartinmichalski@gmail.com>

#### What this PR does / why we need it:
The chart currently lacks readiness probes for sentinels and exporter pods

#### Special notes for your reviewer:
Follow up after following pr:
https://github.com/DandyDeveloper/charts/pull/109

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: charts/redis-ha/values.yaml [hunk 1]
```diff
@@ -284,6 +284,14 @@ sentinel:
     successThreshold: 1
     failureThreshold: 5
 
+  # readiness probe parameters for sentinel container
+  readiness:
```
Comment (DandyDeveloper) [inline charts/redis-ha/values.yaml:288]: `readinessProbe`?

**Review by DandyDeveloper — CHANGES_REQUESTED** [review]
Minor update needed.

File: charts/redis-ha/values.yaml (hunk 1 above)
Comment (michael-michalski) [reply charts/redis-ha/values.yaml]: Thanks for pointing out, ups.

**Comment by michael-michalski** [comment]
@DandyDeveloper Are we happy :D

**Review by DandyDeveloper — APPROVED** [review]

**Pull request merged**